### PR TITLE
A02

### PR DESCRIPTION
**Pull request by Harshini-Jesi**

### CODE REVIEW

**Comment by meenashriTMM** [comment]
Just saw that there are multiple commits in every branch you created. 
Why Training exercise commits are part of Academy exercise?

**Comment by Harshini-Jesi** [comment]
When I want to switch between branches, I just commit the changes and then switch....That's why there are multiple commits mam.

File: Program.cs [hunk 1]
```diff
@@ -1,7 +1,37 @@
...
+      switch (key) {
+         case ConsoleKey.E: return Mode.Easy;
+         case ConsoleKey.M: return Mode.Medium;
+         case ConsoleKey.H: return Mode.Hard;
+         default: Console.WriteLine ("Incorrect key\n"); break;
+      }
+   }
+}
+
+int GetMax (Mode mode) {
```
Comment (meenashriTMM) [inline Program.cs:19]: Try writing this method using switch expression

File: Program.cs [hunk 2]
```diff
@@ -1,7 +1,37 @@
...
+   switch (mode) {
+      case Mode.Easy: return 10;
+      case Mode.Medium: return 100;
+      default: return 1000;
+   }
+}
+Console.WriteLine ($"Guess a number between 1 and {max}\n");
```
Comment (meenashriTMM) [inline Program.cs:26]: It would be more readable if you have your method implementations at the end after main program

**Review by meenashriTMM — CHANGES_REQUESTED** [review]

**Review by meenashriTMM — APPROVED** [review]

**Pull request closed without merging**